### PR TITLE
added fetcher test cases

### PR DESCRIPTION
**Pull request by vardhaman-surana**
## Which issue(s) this PR fixes:

Fixes #

## Checklist:

- [ ] I ran tests as well as code linting locally to verify my changes. 
- [ ] I have done manual verification of my changes, changes working as expected.
- [ ] I have added new tests to cover my changes.
- [ ] My changes follow contributing guidelines of Fission.
- [ ] I have signed all of my commits.

### CODE REVIEW

File: test/e2e/framework/services/services.go [hunk 1]
```diff
@@ -153,3 +143,28 @@ func StartServices(ctx context.Context, f *framework.Framework, mgr manager.Inte
 
 	return nil
 }
+
+func StartStorageSvc(ctx context.Context, f *framework.Framework, mgr manager.Interface) (storageSvcPort int, err error) {
+	storageDir, err := os.MkdirTemp("/tmp", "storagesvc")
+	if err != nil {
+		return 0, fmt.Errorf("error creating temp directory: %v", err)
```
Comment (sanketsudake) [inline test/e2e/framework/services/services.go:150]: Please use %w instead. Same for all error message below

Comment (vardhaman-surana) [reply test/e2e/framework/services/services.go:150]: updated

Comment (vardhaman22) [reply test/e2e/framework/services/services.go]: done

Comment (vardhaman-surana) [reply test/e2e/framework/services/services.go]: done

**Review by sanketsudake — APPROVED** [review]

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**